### DISCUSSION TITLE
Crasher polygon variants

### DISCUSSION THREAD

**Discussion by TheGreenSquare** [Taureon/aps-plus-plus#210] [Suggestions]
A new gamemode modifier was added to Arras.io called "Classic". While it mainly disables the ability to use the level up key, it also adds back Crashers as another polygon variant.

These could be added to the labyrinth polygon set, possibly disabled by default.

**Comment by TheGreenSquare** [comment]
added

**Comment by Taureon** [comment]
iirc i didnt finish crasher variants
